sdl-core/src/main/scala/io/smartdatalake/util/misc/FileUtil.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2023 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.util.misc
+
+import io.smartdatalake.config.ConfigurationException
+import org.apache.hadoop.fs.Path
+
+import java.io.{InputStream, InputStreamReader}
+
+object FileUtil {
```

Comment:
The name is misleading in my opinion. The methods dont handles files, but classpath ressources. Could we name it ResourceUtil?

Reply:
Sounds good to me. :)